tools/mo/openvino/tools/mo/ops/swapaxis.py

```diff
@@ -30,3 +32,11 @@ def infer(node: Node):
             node.out_port(0).data.set_value(np.transpose(node.in_port(0).data.get_value(), axes=node.order))
 
         PermuteAttrs.create_permute_attrs(node, attrs=[('order', 'input:0')])
+
+    @staticmethod
+    def reverse_infer(node: Node):
+        output_shape = node.out_port(0).data.get_shape()
+        if output_shape is not None:
```

Comment:
please insert input_shape is None. it avoids extra copy if input shape is defined

Reply:
done